device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/port_config.ini

```diff
@@ -0,0 +1,41 @@
+# name         lanes             alias        index  role       speed      asic_port_name     coreId      corePortId      numVoq
```

Comment:
Can we remove this file, it is not needed

Reply:
Fixed